lib/Form/Type/FieldType/ImageFieldType.php

```diff
@@ -38,6 +38,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 TextType::class,
                 [
                     'label' => /** @Desc("Alternative text") */ 'content.field_type.ezimage.alternative_text',
+                    'required' => false
```

Comment:
```suggestion
                    'required' => false,
```

Reply:
Forced updated the commit, should be fine now:
https://github.com/ezsystems/repository-forms/pull/330/commits/ce2b95910a2c49a9da97224b10bf1df9342609d2